packages/components/src/components/button-group/style.scss

```diff
@@ -1,6 +1,11 @@
+@import '../@shared/mixins';
 @import '../style';
 
 @layer kol-component {
+	:host {
```

Comment:
Hi Stefan, Du kannst das einfach in der GLOBALS hinterlegen. Dann haben das immer alle Components zuerst.

order
GLOBALS
COMPONENT_STYLE
COMPONENT_THEME_STYLE

Reply:
Hatte ich auch kurz dran gedacht, allerdings wollen wir es nicht in allen Komponenten haben (siehe z.B. Badge und Icon). Jetzt ist es zwar relativ viel Wiederholung, dafür können wir die Font-Size aber auf Komponente-Ebene steuern. Was denkst du?

Reply:
Erwischt :-)